src/data/nat/gcd.lean

```diff
@@ -467,6 +467,16 @@ by simp [coprime]
 @[simp] theorem coprime_self (n : ℕ) : coprime n n ↔ n = 1 :=
 by simp [coprime]
 
+lemma gcd_mul_of_dvd_coprime (a b c : ℕ) (b_dvd_c : b ∣ c) (hac : coprime a c) :
```

Comment:
```suggestion
lemma gcd_mul_of_dvd_of_coprime (a b c : ℕ) (b_dvd_c : b ∣ c) (hac : coprime a c) :
```

Reply:
Arguably, it would be good to swap `b_dvd_c` and `hac`. Because the assumption on `a` would then come before the assumption on `b`. If you do that, please swap the components of the name as well.

Reply:
Edits made. Thank you.